package.json

```diff
@@ -56,7 +56,8 @@
     "request-promise": "1.x",
     "source-map-support": "0.x",
     "teen_process": "1.x",
-    "winston": "2.x"
+    "winston": "2.x",
+    "heapdump": "0.x"
```

Comment:
i think we try to keep this list of deps alphabetized. if it looks alphabetized, mind moving this up?